library/core/registry/src/main/java/org/quiltmc/qsl/registry/mixin/client/ItemRendererAccessor.java

```diff
@@ -0,0 +1,21 @@
+package org.quiltmc.qsl.registry.mixin.client;// Created 2022-04-12T11:27:59
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import net.minecraft.client.color.item.ItemColors;
+import net.minecraft.client.render.item.ItemRenderer;
+
+/**
+ * Allows for ItemColors to be resynchronised with the registry, fixing the bug
+ * of the incorrect color renderer being applied to items due to the backing IDs
+ * being shifted.
+ *
+ * @author KJP12
+ * @since 3.0.0
+ **/
```

Comment:
```suggestion
/**
 * Allows for {@link ItemColors} to be resynchronised with the registry, fixing the bug
 * of the incorrect color renderer being applied to items due to the backing raw identifiers
 * being shifted.
 *
 * @author KJP12
 * @since 3.0.0
 */
```